metadata-auth/apache-ranger-plugin/build.gradle

```diff
@@ -0,0 +1,69 @@
+apply plugin: 'java'
+apply plugin: 'com.github.johnrengelman.shadow'
+
+
+
+compileJava {
+    options.debug = true
+}
+
+shadowJar {
+    determineAndSetVersion()
+    archiveFileName = "$project.name-${version}.jar"
+    exclude "META-INF/*.RSA", "META-INF/*.SF","META-INF/*.DSA"
+}
+
+dependencies {
+    implementation 'org.apache.ranger:ranger-plugins-common:2.2.0'
```

Comment:
This version has a security vulnerability needs to be 2.3.0

Reply:
```
    implementation('org.apache.ranger:ranger-plugins-common:2.3.0') {
        exclude group: "org.apache.htrace", module: "htrace-core4"
    }
    implementation 'org.apache.hadoop:hadoop-common:3.3.4'
```

Reply:
Looks like this also exists in the datahub-ranger-plugin/build.gradle so we need to add the htrace and commons changes. This is the source of quite a few security vulnerabilities. If not addressed here will create a follow-up PR.

Reply:
https://github.com/datahub-project/datahub/pull/6577/files

Reply:
+1